cluster/canary/BUILD.bazel

```diff
@@ -23,6 +23,7 @@ k8s_objects(
         ":summarizer",
         ":updater",
         ":config_merger",
+        "tabulator",
```

Comment:
We should not do this until the release we have in production includes a tabulator image.